queue/group_remote_mongo.go

```diff
@@ -57,6 +63,15 @@ type MongoDBQueueGroupOptions struct {
 	TTL time.Duration
 }
 
+// RegexpMongoDBQueueOptions represents a mapping from a regular expression to
+// match named queues in a queue group to options for those queues.
+type RegexpMongoDBQueueOptions struct {
+	// Regexp is the regular expression to match against the queue ID.
+	Regexp regexp.Regexp
```

Comment:
Would it make more sense to make this a pointer?

Reply:
I guess it could be made a pointer, but it's not an optional field, so I was inclined to not let it be potentially nil.